Лабораторная работа 5/task_1.py

```diff
@@ -0,0 +1,6 @@
+# TODO решить с помощью list comprehension и распечатать его
+from pprint import pprint
+
+dict_ = [{"bin":bin(x),"dec":(x),"hex":hex(x),"oct":oct(x)}for x in range(16)]
```

Comment:
"dec":(x) --> "dec": x
dict_ = [{"bin": bin(x), "dec": x, "hex": hex(x), "oct": oct(x)} for x in range(16)]